style/index.scss

```diff
@@ -9,3 +9,7 @@
 @import './fonts/league-gothic/stylesheet';
 @import 'mapview';
 @import 'header';
+
+* {
```

Comment:
can this be added to another scss file, so the index remains clean only with the imports